Sources/MockableMacro/Requirements/Requirements.swift

```diff
@@ -11,7 +11,8 @@ struct Requirements {
 
     // MARK: Properties
 
-    var syntax: ProtocolDeclSyntax
+    let syntax: ProtocolDeclSyntax
+    let membersModifiers: DeclModifierListSyntax
```

Comment:
I would prefer naming this just "modifiers"

Reply:
Do you think naming it modifiers will not confuse it with the protocol modifiers?